x-pack/test/plugin_api_integration/plugins/oidc_provider/index.js

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
__suggestion:__ I think it makes more sense to have this plugin right in `oidc_api_integration/fixtures/` instead. 

Btw you can also use TypeScript for fixture plugins and integration tests as well :)